File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        if(n == m) {
            if(eq.equals("=")) return 1;
        }
        if(ineq.equals(">")) {
            if(n > m) return 1;
        } else {
            if(n < m) return 1;
        }
        return answer;
    }
}

File: kyunghee/프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    if eq == "=":
        if ineq == "<":
            return 1 if n <= m else 0
        elif ineq == ">":
            return 1 if n >= m else 0
    elif eq == "!":
        if ineq == "<":
            return 1 if n < m else 0
        elif ineq == ">":
            return 1 if n > m else 0